festipals/controllers/ActController.js

```diff
@@ -0,0 +1,11 @@
+var Act = require('../model/pal');
+
+
+  var createUserAccount = function (req, res) {
```

Comment:
well, I was trying to play around with the controller. But I had problems with the routes.